### ISSUE TITLE
Deleting an Interest Creates a Dislike

### ISSUE THREAD

**Issue by larkinds** [larkinds/EthicalTargeting#16]
Deleting an interest item should automatically add that item to the dislike's object and update the UI

<img width="982" alt="Screen Shot 2021-04-12 at 5 20 34 PM" src="https://user-images.githubusercontent.com/41243936/114464229-7b004a80-9bb3-11eb-812e-4c68068d7849.png">

**Comment by jkmartindale** [comment]
I don't think this should be intended behavior. Sometimes there are things that I don't actively have an interest in but don't have an active disinterest in either.

If this were to be implemented, the X button would need to be changed to something that implies a "swap", to better fit with the UI mental model the user already has with other delete buttons they've seen in the past.

**Comment by larkinds** [comment]
Yeah, the X button could trigger a pop-up that asks if a user would like to put the item into dislikes. That'd be suuuuper annoying to do for every item tho. Perhaps a 'select' button that allows you to select multiple items and then either delete en-masse or move to dislikes en masse.

Perhaps best to leave it alone for now 😋
